saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_taxes.py

```diff
@@ -0,0 +1,457 @@
+import json
+from decimal import Decimal
+from functools import partial
+
+from freezegun import freeze_time
+from prices import Money, fixed_discount
+
+from .....core.prices import quantize_price
+from .....discount import DiscountValueType, VoucherType
+from .....graphql.core.utils import to_global_id_or_none
+from .....order.models import Order
+from .....tests.fixtures import recalculate_order
+from .....webhook.event_types import WebhookEventSyncType
+from .....webhook.models import Webhook
+from ...tasks import create_delivery_for_subscription_sync_event
+
+TAXES_SUBSCRIPTION_QUERY = """
```

Comment:
IMO it should go to `saleor/plugins/webhook/tests/subscription_webhooks/subscription_queries.py`, as there are all subscription queries.

Reply:
Hhm, I can move it there, but as far as I understand the idea of this file is to have a common object that could be shared between different subscription. In my case, I have a custom object that will be used only for taxes. Don't you think that in that case, we could keep it in the same file, to have better readability?